src/_test_/user.test.js

```diff
@@ -0,0 +1,19 @@
+import supertest from 'supertest'
+import { MongoMemoryServer } from 'mongodb-memory-server-core'
+import app from '../server.js'
+
+const request = supertest(app)
+
+
+describe('POST /api/register', () => {
+    it('responds with json', async () => {
+        const response = await request.post('/api/register')
+        .send({email: 'testdev@dev.com', password: '1234567'})
+        .set('Accept', 'application/json')
```

Comment:
Codacy has a fix for the issue: [Replace `········.set('Accept',·'application/json')` with `······.set('Accept',·'application/json');`](https://app.codacy.com/gh/Denniman/group-four-BE/pullRequest?prid=7801432)
```suggestion
      .set('Accept', 'application/json');
```